containers/deploy/secrets/import_secrets.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: Create Import Secret
+  k8s_v1_secret:
+      state: "{{ deployment_state }}"
+      force: false
+      resource_definition:
+          apiVersion: v1
+          kind: Secret
+          metadata:
+              name: data-import
+              namespace: "{{ project_name }}"
+          type: Opaque
+      data:
+          id_rsa: '{{ import_key | b64encode }}'
```

Comment:
I am missing where this ends up getting used via a volumeMount.

Reply:
https://github.com/ehelms/forklift/pull/18/files#diff-795fb7a5c61e5ce8d7524ac21b2b2f7bR48